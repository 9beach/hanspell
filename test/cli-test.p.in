(우리 집) 으로 간다.
