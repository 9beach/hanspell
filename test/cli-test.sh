#!/usr/bin/env bash

set -e

cat test/cli-test.in | lib/cli.js -d 2>&1 | diff - test/cli-test.out.d
cat test/cli-test.in | lib/cli.js -p 2>&1 | diff - test/cli-test.out.p
cat test/cli-test.in | lib/cli.js -j 2>&1 | diff - test/cli-test.out.j
cat test/cli-test.in | lib/cli.js -a 2>&1 | diff - test/cli-test.out.a
cat test/cli-test.p.in | lib/cli.js -p 2>&1 | diff - test/cli-test.p.out
cat test/cli-test.d.in | lib/cli.js -d 2>&1 | diff - test/cli-test.d.out
