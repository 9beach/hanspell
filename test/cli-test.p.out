# hanspell-cli: (우리 집) 으로 -> (우리 집)으로
(우리 집)으로 간다.
